library/DataFixtures/ORM/ProviderCompany.php

```diff
@@ -183,6 +183,7 @@ public function load(ObjectManager $manager)
             $this->setCountry($fixture->getReference('_reference_ProviderCountry70'));
             $this->setTransformationRuleSet($fixture->getReference('_reference_ProviderTransformationRuleSet70'));
             $this->setVoicemailNotificationTemplate($fixture->getReference('_reference_ProviderNotificationTemplate1'));
+            $this->setCorporation($fixture->getReference('_reference_Corporation1'));
```

Comment:
Corporations are currently only used for intervpbx Friends so only vPBX companies make use of having a Corporation assigned. This Fixture is assigning a corporation to a Wholesale type client, which could be handled like an error. We will discuss if we want to validate only vPBX companies can have coporation assigned.

Reply:
The final decission is to validate in Company sanitizer that type is vpbx if corporationId has value. In other words, only vPBX Companies can have corporationId with value.


It would be great to change also fixtures to be coherent with this decision.